swap/src/cli/command.rs

```diff
@@ -53,6 +56,9 @@ pub enum Command {
 
         #[structopt(long = "tor-socks5-port", help = "Your local Tor socks5 proxy port", default_value = DEFAULT_TOR_SOCKS5_PORT)]
         tor_socks5_port: u16,
+
+        #[structopt(long = "bitcoin-target-block", help = "Within how many blocks should the Bitcoin transactions be confirmed.", default_value = DEFAUL_BITCOIN_CONFIRMATION_TARGET)]
```

Comment:
Since this is suplicated for multiple commands you might want to extract that into a separate struct and use [`flatten`](https://docs.rs/structopt/0.3.21/structopt/#flattening).

Reply:
(same accounts for some other params already in there - we can also clean this up in a follow-up)

Reply:
Created a follow-up issue: #478